frontend/src/services/api/index.ts

```diff
@@ -37,9 +38,12 @@ api.interceptors.response.use(
   },
   (error) => {
     if (error.response?.status === 403) {
+      const allCookies = Cookies.get(); // Get all cookies
+      for (const cookie in allCookies) {
+        Cookies.remove(cookie); // Remove each cookie
+      }
       router.push({
         name: "login",
-        params: { next: router.currentRoute.value.path },
```

Comment:
why remove this? it's what brings users back to the page they were just on after login

Reply:
it was not working for me and tried to fix it but forgot about it. Fixing it now!